src/cleanfigure.m

```diff
@@ -139,7 +141,14 @@ function pruneOutsideBox(meta, handle)
 
   xData = get(handle, 'XData');
   yData = get(handle, 'YData');
-  zData = get(handle, 'ZData');
+
+  % Obtain zData, if available
+  handleFields = get(handle);
+  if isfield(handleFields, 'ZData')
+    zData = get(handle, 'ZData');
+  else
+    zData = [];
+  end
```

Comment:
I personally prefer a `try/catch` in these cases:

```matlab
  try
      zData = get(handle, 'ZData');
  catch
      zData = [];
  end
```
However, since I think we are in disagreement, I would suggest to use directly `isprop(handle,'ZData')` without qu
erying all properties `handleFields = get(handle);`.